LIMS2DB/flowcell_sql.py

```diff
@@ -17,10 +17,14 @@ def create_lims_data_obj(session, pro):
              inner join containerplacement cp on cp.containerid=ct.containerid \
              inner join processiotracker piot on piot.inputartifactid=cp.processartifactid \
              where piot.processid = {pid}::integer;".format(pid=pro.processid)
-    
+
     cont=session.query(Container).from_statement(text(query)).first()
     obj['container_id']=cont.luid
     obj['container_name']=cont.name
+    try:
+        obj['container_running_notes']=cont.udfs[0].udfvalue
```

Comment:
I would like to object against general try except clauses where the specific exception is not specified... I think there is a risk of introducing silent bugs this way...

Reply:
Well we cant (or we shouldn't) `raise` an error here, as this script is intended to run as *cronjob*. So if one container/flowcell has some problem fetching running notes then it would raise and not process the remaining containers.

But I see your point too @alneberg , so maybe instead of silently moving on we could log the error and move on. That being said the only error I could think of that can happen (from my suggestion) is `TypeError`, when `cont.udfs` is not a iterable object or `cudf.udfvalue` is not a `dict` (both are highly unlikely)

Reply:
I see. I didn't get the exact context. I like the idea of logging. Also if we would check for both of those errors explicitly it would be much easier to understand the potential problems for someone rewriting this code in five years from now.